client/src/components/EditCommentPopup.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react'
+import Modal from 'react-bootstrap/Modal'
+import Button from 'react-bootstrap/Button'
+import TextArea from './TextArea'
+
+export default function EditCommentPopup({show, onClose, text, rating, comment_id, commentUpdated}) {
+
+    const [editedComment, setEditedComment] = useState({
+        "text" : text,
+        "rate" : rating,
+        "comment_id" : comment_id
+    })
```

Comment:
If you ever find yourself repeating the key names with the variable name, you can simplify writing that object like so,

```suggestion
    const [editedComment, setEditedComment] = useState({
        text,
        rating,
        comment_id
    })
```

When a variable is used in an object, and it does not have a key, then JS will use the variable name as the key.

Reply:
The incredible power of JS! Thank you, got it!